### ISSUE TITLE
Not an actual issue but thanks for being hero we need but dont deserve

### ISSUE THREAD

**Issue by kir12** [Xmader/musescore-downloader#8]
real scummy move on musescore.com's part to suddenly make the entirety of downloading paywalled

**Comment by trevortrusty** [comment]
Agreed. I'm not okay with me putting my arrangements I worked hard to score onto the website if they are going to make people pay for it while I'm doing it for free. Makes no logical sense, and I will be using this script in my browser from now on.

**Comment by quizilkend** [comment]
Totally agreed!

**Comment by answer2question** [comment]
thank you foe providing a resource that makes music free for everybody. you are a GOAT.

**Comment by zyf722** [comment]
many thanks for the script to the developers

**Comment by jeetee** [comment]
> Agreed. I'm not okay with me putting my arrangements I worked hard to score onto the website if they are going to make people pay for it while I'm doing it for free. Makes no logical sense, and I will be using this script in my browser from now on.

@trevortrusty How about marking your work as original work then, thus making them freely available again?
I agree that copyright legislation doesn't always make sense; but publishing an arrangement is not allowed without consent of the copyright holder.
This isn't musescore.com rule, it is law.

**Comment by mosest** [comment]
You really are the hero we need but don't deserve. This script works awesome on Chrome, and it's so easy to use. Thanks so so much for putting in the effort. <3

**Comment by Poshtester63** [comment]
Leaving a comment here to thank you for your work on this script! :D

**Comment by trevortrusty** [comment]
> 
> 
> > Agreed. I'm not okay with me putting my arrangements I worked hard to score onto the website if they are going to make people pay for it while I'm doing it for free. Makes no logical sense, and I will be using this script in my browser from now on.
> 
> @trevortrusty How about marking your work as original work then, thus making them freely available again?
> I agree that copyright legislation doesn't always make sense; but publishing an arrangement is not allowed without consent of the copyright holder.
> This isn't musescore.com rule, it is law.

I see what you're saying, but is posting sheet music to musescore _really_ publishing? I can't imagine posting a mscz file to the musescore platform being the same thing as publishing your sheet music on musicnotes.com for example.

**Comment by jeetee** [comment]
> is posting sheet music to musescore _really_ publishing? I can't imagine posting a mscz file to the musescore platform being the same thing as publishing your sheet music on musicnotes.com for example.

It is in they eye of the law; publishing a score means making it publicly available, regardless of whether you monetize that or not.

**Comment by lordantonio101** [comment]
Thankyou so much for this! Good work!

**Comment by Mabinogi-delight** [comment]
作者我爱死你了！！

**Comment by Lee-Yu-Chen** [comment]
Xmander先生/女士，请问有什么方法可以代表我对您的感谢？

**Comment by Mabinogi-delight** [comment]
@jeetee That's why the entire copyright law thing is BS. An archeologist can spent years trying to recover text from ancient tomb, yet his work is not protected by copyright. However, someone taking a picture of the very same tomb can get copyrighted.

**Comment by jeetee** [comment]
@Mabinogi-delight 
> That's why the entire copyright law thing is BS. 

I don't disagree, but I also don't think MuseScore is to blame for the current law. Especially when they've put in the effort to protect all uploaders from prosecution and still allow everyone to view/playback many of those protected scores for free.


> An archeologist can spent years trying to recover text from ancient tomb, yet his work is not protected by copyright. However, someone taking a picture of the very same tomb can get copyrighted.

His work would be the recovery of the text, not the text itself. Since he did not "create" something it indeed can't get copyrighted. The photo can be considered an original work of art considering its composition, which indeed can get copyrighted. But that copyright can only hold if the composition itself is unique enough, just a random smartphone snapshot of the tomb is not protected by copyright law, as there again (just as with the archaeologist recovery) no original creative content has been created.

**Comment by nazmifr** [comment]
+10²²²²²²²²² 

All hail to Xmander our savior!

**Comment by Xmader** [comment]
Er... I'm Xmader without the n in the middle.My name is not an actual word, or the compound of two words. I picked the name by hitting my keyboard randomly back in 2015.

**Comment by bucketohwater** [comment]
Thanks for this, this is amazing!

**Comment by ajlee2006** [comment]
Thank you very much Xmader for your work.

**Comment by Darkhellkaiser** [comment]
Totally agreed! 
Thank you very much Xmader
I really appreciate your help.
Thank you again!

**Comment by joybod** [comment]
i was using musescore for playing music in video games for the meme, now I once again can spam elder scolls and orchestra music in space station 13
